Felienne/hedy
# Implemented a 403 error page

Provide a general summary of your changes in the Title of the PR

**Description**

Fixes #1161. Adds a dedicated 403 page and image for the html page (as a note, the current image used was something I put together based on the current image used and general 403 images. Let me know if another image was in mind)

**Fix for**

For for #1161

**How to test**

1. Set up testing environment and login (some links only show the 403 page when the user is logged in, otherwise the user is redirected to the login page)
2. Visit the following links and observe the page:

Only teachers can retrieve classes
http://localhost:5000/classes
http://localhost:5000/class/foo
http://localhost:5000/customize-class/foo

Looks like you are not a teacher!
http://localhost:5000/programs?user=foo
http://localhost:5000/for-teachers
